lib/workers/pr/body/common.ts

```diff
@@ -14,16 +14,18 @@ export type PrBodyUpgrade = LookupUpdate &
   };
 
 // TODO: Unify and optimize types and interfaces
-export type PrBodyConfig<T extends PrBodyUpgrade = PrBodyUpgrade> = {
-  upgrades: T[];
-  prBanner?: string;
-  prBodyTemplate: string;
-  prBodyColumns: string[];
-  prBodyDefinitions: Record<string, string>;
-  prTitle?: string;
-  updateType: string;
-  isGroup?: boolean;
-  isPin?: boolean;
-  hasReleaseNotes?: boolean;
-  global?: { prBanner?: string; prFooter?: string };
-} & Partial<RenovateSharedConfig>;
+export type PrBodyConfig = BranchConfig;
+
+// = {
+//   upgrades: T[];
+//   prBanner?: string;
+//   prBodyTemplate: string;
+//   prBodyColumns: string[];
+//   prBodyDefinitions: Record<string, string>;
+//   prTitle?: string;
+//   updateType: UpdateType;
+//   isGroup?: boolean;
+//   isPin?: boolean;
+//   hasReleaseNotes?: boolean;
+//   global?: GlobalConfig;
+// } & Partial<RenovateSharedConfig>;
```

Comment:
Delete?

Reply:
ups, sure. maybe i can remove some more obsolete interfaces and reexported type declarations